geoportailv3/static/js/infobar/elevationdirective.js

```diff
@@ -25,7 +25,8 @@ app.elevationDirective = function() {
   return {
     restrict: 'E',
     scope: {
-      'map': '=appElevationMap'
+      'map': '=appElevationMap',
+      'visible': '=appVisible'
```

Comment:
I'd suggest using `appElevationVisible` or `appElevationActive`. The latter is what we use for the measure directive.